content/blog/2025-02-28-conf-info-event.md

```diff
@@ -0,0 +1,91 @@
++++ title = "Join Us at Nordic-RSE conference 2025: A Home for Research Software Enthusiasts!"
+
+[extra]
+
+author = "Samantha Wittke" +++
+
+> Have you ever felt like the work you do in research software goes unnoticed? Like the infrastructure, tools,
+> and coding behind scientific breakthroughs don’t always get the recognition they deserve? You’re not alone—and that’s exactly why this conference exists!
+
+The Nordic-RSE conference is an event designed for anyone passionate about research software engineering (RSE). 
+Whether you develop research tools, support scientific software, or are simply curious about RSE, 
+this is the perfect space to learn, connect, and share knowledge. On February 27th we met with the 
+conference chair Richard Darst and local organizer Matteo Tomasini to chat about the upcoming conference.  
+
+#### What is RSE and Why Should You Care?
+
+Research Software Engineering is the backbone of modern research. It’s the coding, infrastructure, and tools 
+that enable scientific discoveries—but often, it’s undervalued in academia.
+
+As one of our interviewees put it: “RSE is everything research-related that isn’t just publishing papers. 
+It’s about the software, the infrastructure, the tools that make research possible.”
+
+Another described RSE as “a space for people who care about software and everything around it.” 
+This conference is about celebrating that space and bringing together a community that understands the importance of research software.
+
+#### What Makes This Conference Different?
+
+Unlike traditional domain-specific conferences, this one is about:
+
+- Techniques over results – It’s not about presenting research findings but about sharing the methods,
+- tools, and best practices that make research possible.
```

Comment:
There is a line break here that shouldn't be here. It creates an extra item (same for line 33)